src/forms/fields/internal/TitleField.jsx

```diff
@@ -50,8 +57,8 @@ const TitleField = ({
   if (!title) return null;
 
   const titleText = (
-    <Typography.Text
-      style={{ fontSize: isObject && "12pt" }}
+    <Typography.Paragraph
```

Comment:
why change?

Reply:
The html element generated by `Text` has the same width as the text, so you need to hover over it to reveal the anchor icon. With `Paragraph` it expands to fill all the row, so the icon would be revealed by hovering anywhere on the row, which I like more.